linux-bash/read/src/main/bash/read_inputs.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# section 2.1
+default_read() {
+    read input1 input2 input3
+    echo "[$input1] [$input2] [$input3]"
+}
+
+# section 2.2
+custom_ifs_no_array(){
```

Comment:
A minor point, let's be consistent with the spaces and parenthesis like you have been with some of the other functions. ;-)

Reply:
done